elastic/kibana
# [Fix] Lose OriginatingApp Connection on Save As

## Summary

Closes https://github.com/elastic/kibana/issues/72429 by restoring the flow from 7.8. When a user has redirected to an editor from a container, then used the 'save as' button and chosen **not** to return to the originating app, the connection to that app should be severed.

### How to test

- Edit a panel from a dashboard OR create a new panel from dashboard
- Once you are in the editor, use the modal (save as), to save the visualization / lens visualization
- Deselect `add to dashboards after saving` select `save as new visualization` then save.
- The originatingApp connection should now be lost, and the top nav menu should only display `save`

### Checklist

Delete any items that are not applicable to this PR.


~~- [ ] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md)~~
~~- [ ] [Documentation](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#writing-documentation) was added for features that require explanation or tutorials~~
~~- [ ] This was checked for [keyboard-only and screenreader accessibility](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility#Accessibility_testing_checklist)~~
~~- [ ] This renders correctly on smaller devices using a responsive layout. (You can test this [in your browser](https://www.browserstack.com/guide/responsive-testing-on-local-server)~~
~~- [ ] This was checked for [cross-browser compatibility](https://www.elastic.co/support/matrix#matrix_browsers)~~

### For maintainers
